lib/rules/filename-blocklist.js

```diff
@@ -31,17 +35,23 @@ export default {
           type: 'string',
         },
       },
+      {
+        stringResponse: {
+          type: 'boolean',
+        },
+      },
```

Comment:
Would it be possible to make the new `nonGlob` parameter a property of an object, making it easier for future extensions?
```js
{
  code: "var foo = 'bar';",
  filename: 'src/foo.models.ts',
  options: [
    {
      '*.models.ts': 'it is a generic naming convention. Please see http://someExample.com for our repo's rules and naming suggestions ',
    },
    {
      nonGlob: true //optional argument that allows the value string to not be checked as a globbing pattern
    }
  ],
}
```

You can refer this: https://github.com/dukeluo/eslint-plugin-check-file/blob/c5e7e89ee6daaf44277347b3c48ca070a9854415/lib/rules/filename-naming-convention.js#L67

Reply:
Maybe the variable name like `nonGlobSuggestion` will be better.